.env

```diff
@@ -0,0 +1 @@
+MONGODB_URI=mongodb+srv://angekarara2022:zLjp9jQK54MZBgoB@cluster0.tkgr71g.mongodb.net/todos?retryWrites=true&w=majority&appName=Cluster0
```

Comment:
☠️ ☠️ It's not a good practice to push .env file due to the way it contains our secret tokens kindly consider reading more on why .env 😃  kindly consider adding it in .gitignore